src/Orleans/Serialization/DeserializationContext.cs

```diff
@@ -27,12 +27,12 @@ MIT License
 
 namespace Orleans.Serialization
 {
-    internal class DeserializationContext
+    public class DeserializationContext
     {
         [ThreadStatic]
         private static DeserializationContext ctx;
 
-        internal static DeserializationContext Current
+        public static DeserializationContext Current
```

Comment:
This is referenced from generated code in order to record the object currently being deserialized.